composer.json

```diff
@@ -17,7 +17,8 @@
     "require": {
         "php": ">=7.1",
         "symfony/polyfill-mbstring": "^1.18",
-        "ext-zlib": "*"
+        "ext-zlib": "*",
+        "ext-iconv": "*"
```

Comment:
Why did you add this line?

Reply:
Because `iconv()` method from iconv extension used during converting pdf-file encoding to utf-8 in [Font class] (https://github.com/smalot/pdfparser/pull/500/files#diff-27bc594b24bd5e2779e8d81ee79810d0ffda03f200f0d53be007a44a9d2cb2deR409)